src/nvme/types.h

```diff
@@ -1485,27 +1485,35 @@ enum nvme_id_ctrl_cqes {
 /**
  * enum nvme_id_ctrl_oncs - This field indicates the optional NVM commands and
  *			    features supported by the controller.
- * @NVME_CTRL_ONCS_COMPARE:		If set, then the controller supports
- *					the Compare command.
- * @NVME_CTRL_ONCS_WRITE_UNCORRECTABLE:	If set, then the controller supports
- *					the Write Uncorrectable command.
- * @NVME_CTRL_ONCS_DSM:			If set, then the controller supports
- *					the Dataset Management command.
- * @NVME_CTRL_ONCS_WRITE_ZEROES:	If set, then the controller supports
- *					the Write Zeroes command.
- * @NVME_CTRL_ONCS_SAVE_FEATURES:	If set, then the controller supports
- *					the Save field set to a non-zero value
- *					in the Set Features command and the
- *					Select field set to a non-zero value in
- *					the Get Features command.
- * @NVME_CTRL_ONCS_RESERVATIONS:	If set, then the controller supports
- *					reservations.
- * @NVME_CTRL_ONCS_TIMESTAMP:		If set, then the controller supports
- *					the Timestamp feature.
- * @NVME_CTRL_ONCS_VERIFY:		If set, then the controller supports
- *					the Verify command.
- * @NVME_CTRL_ONCS_COPY:		If set, then the controller supports
- *					the copy command.
+ * @NVME_CTRL_ONCS_COMPARE:		  If set, then the controller supports
+ *					  the Compare command.
+ * @NVME_CTRL_ONCS_WRITE_UNCORRECTABLE:	  If set, then the controller supports
+ *					  the Write Uncorrectable command.
+ * @NVME_CTRL_ONCS_DSM:			  If set, then the controller supports
+ *					  the Dataset Management command.
+ * @NVME_CTRL_ONCS_WRITE_ZEROES:	  If set, then the controller supports
+ *					  the Write Zeroes command.
+ * @NVME_CTRL_ONCS_SAVE_FEATURES:	  If set, then the controller supports
+ *					  the Save field set to a non-zero
+ *					  value in the Set Features command and
+ *					  the Select field set to a non-zero
+ *					  value in the Get Features command.
+ * @NVME_CTRL_ONCS_RESERVATIONS:	  If set, then the controller supports
+ *					  reservations.
+ * @NVME_CTRL_ONCS_TIMESTAMP:		  If set, then the controller supports
+ *					  the Timestamp feature.
+ * @NVME_CTRL_ONCS_VERIFY:		  If set, then the controller supports
+ *					  the Verify command.
+ * @NVME_CTRL_ONCS_COPY:		  If set, then the controller supports
+ *					  the copy command.
+ * @NVME_CTRL_ONCS_COPY_SINGLE_ATOMICITY: If set, then the write portion of a
```

Comment:
Please submit the format change of the above lines in a separate commit.  You are only adding two new arguments as a part of this change and it would be best to eliminate all unneeded white space changes from your MR.